src/components/Details/DetailedTrafficList.tsx

```diff
@@ -145,7 +145,7 @@ class DetailedTrafficList extends React.Component<DetailedTrafficProps> {
       }
     } else if (NodeType.SERVICE === node.type) {
       icon = <Icon type="pf" name="service" style={style} />;
-      if (!node.isServiceEntry || !node.isInaccessible) {
+      if (!node.isServiceEntry && !node.isInaccessible) {
```

Comment:
This doesn't look right to me.  Are you sure about this?

Reply:
I think it's right. The `SummaryLink.isInaccessible` is:
```js
const isInaccessible = (data: NodeData): boolean => {
  return data.isInaccessible || data.isServiceEntry !== undefined;
};
```

It is the same condition, but negated - because I want the links if accessible.